src/Servers/Kestrel/Transport.Quic/test/QuicConnectionListenerTests.cs

```diff
@@ -115,8 +115,7 @@ public async Task ClientCertificate_Required_NotSent_ConnectionAborted()
 
             // https://github.com/dotnet/runtime/issues/57246 The accept still completes even though the connection was rejected, but it's already failed.
             var serverContext = await connectionListener.AcceptAndAddFeatureAsync().DefaultTimeout();
-            qex = await Assert.ThrowsAsync<QuicException>(() => serverContext.ConnectAsync().DefaultTimeout());
-            Assert.Equal("Failed to open stream to peer. Error Code: INVALID_STATE", qex.Message);
+            await Assert.ThrowsAsync<QuicException>(() => serverContext.ConnectAsync().DefaultTimeout());
```

Comment:
What's the new exception?

Reply:
I don't remember exactly. It was something like "Stream creation error BLAH BLAH BLAH" 😄